src/applications/check-in/pre-check-in/pages/Introduction/IntroductionDisplay.jsx

```diff
@@ -64,16 +86,8 @@ const IntroductionDisplay = props => {
       <a
         className="vads-c-action-link--green"
         href="#answer"
-        onKeyDown={useCallback(e => {
-          if (e.key === ' ') {
-            e.preventDefault();
-            goToNextPage();
-          }
-        }, [])}
-        onClick={useCallback(e => {
-          e.preventDefault();
-          goToNextPage();
-        }, [])}
+        onKeyDown={handleStart}
```

Comment:
You might be able to get rid of the onKeyDown here. I think having both will result in the handleStart being called twice when you hit the enter key because it registers both as a click and a key down.

Reply:
This is here to handle activation on spacebar press. Which was an older accessibility requirement.